### PR TITLE
Update pytests

### LINKED ISSUE #3: Improvements to pytest suite
Add to pytests based on comments in #2

### PR DESCRIPTION
**Pull request by kushalbakshi**
This PR adds the suggestions from #2 that were not included when that PR was merged. 

To provide clarity on [this comment](https://github.com/datajoint/workflow-zstack/pull/2/files/ad4e20633f44f90fd23a72468473095a1f769c96#r1199389883) from #2: when `diameter` is set to `None`, `cellpose` runs much quicker than if it is set to a specific value

### CODE REVIEW

**Comment by kabilar** [comment]
> To provide clarity on [this comment](https://github.com/datajoint/workflow-zstack/pull/2/files/ad4e20633f44f90fd23a72468473095a1f769c96#r1199389883) from #2: when `diameter` is set to `None`, `cellpose` runs much quicker than if it is set to a specific value

Thanks @kushalbakshi.  It makes sense to set `diameter=None`.  After further review of the documentation, the diameter is automatically estimated when it is set to `None`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,13 @@
...
+## [0.1.1] - 2023-08-31
+
++ Update - Incorporate suggestions from previous PR to improve pytests
+
 ## [0.1.0] - Unreleased
 
 + Add - Workflow pipeline, pytests, CICD
 
 [0.1.0]: https://github.com/datajoint/workflow-zstack/releases/tag/0.1.0
+[0.1.1]: https://github.com/datajoint/workflow-zstack/releases/tag/0.1.1
```
Comment (kabilar) [inline CHANGELOG.md]:
```suggestion
[0.1.1]: https://github.com/datajoint/workflow-zstack/releases/tag/0.1.1
[0.1.0]: https://github.com/datajoint/workflow-zstack/releases/tag/0.1.0
```

**Review by kabilar — APPROVED** [review]
Thanks @kushalbakshi.  All changes look good.  Were you able to successfully run these tests locally?

**Comment by kushalbakshi** [comment]
> Thanks @kushalbakshi. All changes look good. Were you able to successfully run these tests locally?

I have not run the tests locally yet. With the `tutorial-db` offline, I currently don't have access to an account that has permissions to create schemas.

**Comment by kabilar** [comment]
> > Thanks @kushalbakshi. All changes look good. Were you able to successfully run these tests locally?
> 
> I have not run the tests locally yet. With the `tutorial-db` offline, I currently don't have access to an account that has permissions to create schemas.

Let's hold off on merging until these tests have been run locally.  Were you able to get permission to create schemas on `db.datajoint.com`?

**Comment by kushalbakshi** [comment]
> > > Thanks @kushalbakshi. All changes look good. Were you able to successfully run these tests locally?
> > 
> > 
> > I have not run the tests locally yet. With the `tutorial-db` offline, I currently don't have access to an account that has permissions to create schemas.
> 
> Let's hold off on merging until these tests have been run locally. Were you able to get permission to create schemas on `db.datajoint.com`?

I'm in the process of setting one up. Instead of <username_> as the database prefix, in order to maintain consistency with the internal <org-name>_<workflow-name> conventions, we're setting up a test workflow for the sciops team that will retain this structure and have permissions to create schemas

**Comment by kushalbakshi** [comment]
Here is the output of the pytest after the most recent changes:

image

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,8 +3,13 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.1.1] - 2023-08-31
+
++ Update - Incorporate suggestions from previous PR to improve pytests
```
Comment (ttngu207) [inline CHANGELOG.md:8]:
```suggestion
+ Update - Improve pytests
```

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**